internal/utils/csi.go

```diff
@@ -119,6 +121,27 @@ var EtcSelinuxVolume = corev1.Volume{
 	},
 }
 
+func LogDirVolume(LogHostPath, CsiComponentName string) corev1.Volume {
```

Comment:
Why do we need the csi component name as part of the volume name?
I don't see a need to mount a separate volume for each component.

You might want to use different folders inside the same mount, but not different volumes